solr/docker/templates/Dockerfile.official.header.template

```
@@ -85,5 +85,6 @@ RUN set -ex; \
   { command -v gpgconf; gpgconf --kill all || :; }; \
   rm -r "$GNUPGHOME"; \
   tar -C /opt --extract --preserve-permissions --file "/opt/solr-$SOLR_VERSION.tgz"; \
-  rm "/opt/solr-$SOLR_VERSION.tgz"*;
+  rm "/opt/solr-$SOLR_VERSION.tgz"*; \
+  apt -y remove wget gpg dirmngr && apt -y autoremove;
```

No need to remove wget here when we install it again later. Use `apt-get` instead of `apt` for consistency within the file.
```suggestion
  apt-get -y remove gpg dirmngr && apt-get -y autoremove;
```